pkgs/development/tools/jq/default.nix

```diff
@@ -12,7 +13,17 @@ stdenv.mkDerivation rec {
 
   outputs = [ "bin" "doc" "man" "dev" "lib" "out" ];
 
-  buildInputs = [ oniguruma ];
+  # inspired by app-misc/jq/jq-1.6-r3.ebuild from Gentoo
+  postPatch = ''
+    rm -fr ./modules/oniguruma
```

Comment:
Should removal be unconditional?

Reply:
Not important, but I prefer it this way. This way I can be sure that build system does not use vendored version of oniguruma.

Reply:
```suggestion
    rm -r ./modules/oniguruma
```

Reply:
@SuperSandro2000 applied.